App/src/Command/HandleBalanceCommand.php

```diff
@@ -25,6 +32,70 @@ protected function configure(): void
 
     protected function execute(InputInterface $input, OutputInterface $output): int
     {
+        $io = new SymfonyStyle($input, $output);
+        $scenario = $io->choice(
+            'Selectionner le scénario à executer:',
+            [
+                "First",
+                "Second",
+                "Third",
+                "Fourth"
+            ]
+        );
+
+        $datas = match ($scenario) {
```

Comment:
```suggestion
        $data = match ($scenario) {
```
no need to add plurial tense to data :slightly_smiling_face:

Reply:
I passed $datas in a foreach loop , I think with the purial, there is more meaning but i really don't know if i'm right so I will do as you say :)